docs/mkdocs/documentation/deploy_kmod.md

```diff
@@ -60,6 +60,9 @@ Once loaded, kernel modules have all possible permissions to do any kind of oper
 KMM creates privileged workload to load the kernel modules on nodes.
 That workload needs `ServiceAccounts` able to use the `privileged` `SecurityContextConstraint` (SCC).
```

Comment:
```suggestion
[Pod Security admission](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html)
and `SecurityContextConstraints` (SCCs) restrict privileged workload in most namespaces by default; namespaces that are
part of the [cluster payload](https://docs.openshift.com/container-platform/4.12/authentication/understanding-and-managing-pod-security-admission.html#security-context-constraints-psa-opting_understanding-and-managing-pod-security-admission)
are an exception to that rule.  
In namespaces where Pod Security admission and SCC synchronization are enabled, the KMM workload needs to be manually
allowed through RBAC.
This is done by configuring a ServiceAccount that is allowed to use the `privileged` SCC in the `Module`.

The authorization model depends on the `Module`'s namespace, as well as its spec:
```

Reply:
This is a good change, I would change one part though:
```
In namespaces where Pod Security admission and SCC synchronization are enabled, the KMM workload needs to be manually
allowed through RBAC.
```
Changed to:
```
In namespaces where Pod Security admission and SCC synchronization are enabled, Openshift will modify the Pod Security rules base of the most privileged ServiceAccount used in the namespace, therefore, the KMM workload should use a powerful enough ServiceAccount.
```

Reply:
I find this more confusing.

Reply:
Changed as you suggested.